sematic/examples/add/__main__.py

```diff
@@ -1,15 +1,28 @@
 # Standard Library
+import argparse
 import logging
 
 # Sematic
+from sematic import CloudResolver, LocalResolver
 from sematic.examples.add.pipeline import pipeline
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser("Sematic add example")
+    parser.add_argument("--cloud", type=bool, action="store_true", default=False)
+    parser.add_argument("--attach", type=bool, action="store_true", default=False)
```

Comment:
This is the simplest pipeline and the entry point for users. Please keep the default behavior using `LocalResolver`.

Reply:
We need to use this pipeline for testing, we can't add and remove the `CloudResolver` in each PR.
Additionally, this pipeline is not included in the pip wheel, so users do not use this pipeline. Only people cloning the code base and using Bazel to run it.